.changeset/tough-students-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+@nomicfoundation/edr: minor
```

Comment:
Thanks, the missing quotes issue was fixed. 

Re minor change: I prefixed the commit with "feat", because it felt that it better communicated the change than "fix" as there was nothing broken, we just changed to a better approach. And "feat" implies "minor" bump in the [semver](https://www.conventionalcommits.org/en/v1.0.0/) spec.

In any case, I'm happy to change it to "fix" and "patch".

Wdyt @fvictorio ?

Reply:
Ok thanks, changed to patch. Ok to keep "feat" in the prefix?

Reply:
Yes, I'm ok with `feat`. Alternatively we can use a `perf` prefix. I don't have a preference.

Reply:
If we're using the conventional commit standard for prefixes, this would be a `perf:`